scripts/Muon/GUI/Common/utilities/run_string_utils.py

```diff
@@ -71,14 +71,15 @@ def validate_run_string(run_string):
     return False
 
 
-def run_string_to_list(run_string, max_value = True):
+def run_string_to_list(run_string, max_value=True):
     """
     Does the opposite of run_list_to_string(), taking a string representation of a series of runs
     and producing an ordered list of unique runs. Calls validate_run_string().
     :param run_string: string, a series of runs
     :max_value: if to use the max number of runs
     :return: list of integers
     """
+    run_string = run_string.replace(" ", "")
```

Comment:
this makes me nervous as the code is shared and it could break code that is expecting the white space to be present.

Reply:
Should i create one for loading data as this issue also comes up in loading in other muon interfaces if you try loading more than one run with a space for example in the elemental analysis gui 2695- 6 instead of 2695-6 it raises an error and it shos up in all muon interfaces.

Reply:
How do you produce the issue in the other interfaces?

Reply:
when loading multiple runs put a space in between the hyphen and number eg. xxxxxx- x or xxxxxx- xx

Reply:
I loaded multiple runs earlier in the muon analysis GUI and I did not get an error